docs/ops/state/checkpoints.zh.md

```diff
@@ -26,69 +26,49 @@ under the License.
 * toc
 {:toc}
 
-## Overview
+## 概述
+checkpoint使Flink的状态具有良好的容错性，通过checkpoint机制，Flink可以对作业的状态和计算位置进行恢复，因此Flink作业具备高容错执行语义。
 
-Checkpoints make state in Flink fault tolerant by allowing state and the
-corresponding stream positions to be recovered, thereby giving the application
-the same semantics as a failure-free execution.
+参考[Checkpointing]({{ site.baseurl }}/dev/stream/state/checkpointing.html)查看如何在Flink程序中开启和配置checkpoint。
 
-See [Checkpointing]({{ site.baseurl }}/dev/stream/state/checkpointing.html) for how to enable and
-configure checkpoints for your program.
+## 保留Checkpoint
```

Comment:
```suggestion
## 保留 Checkpoint
```

Reply:
@klion26 thank you ,can you close the request,and I push new request?